client/src/components/Typography.tsx

```diff
@@ -23,6 +23,17 @@ const secondaryMainTextStyle = (theme: Theme) =>
     },
   });
 
+const greyBackgroundTextStyle = (theme: Theme) =>
```

Comment:
could the properties `borderRadius`, `margin`, `padding`, and `width` be moved to inline styling? it would make `GreyBackgroundTextTypography` more resuable